model/value_marshal.go

```diff
@@ -0,0 +1,110 @@
+// Copyright 2013 The Prometheus Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package model
+
+import (
+	"math"
+	"strconv"
+	"unsafe"
+
+	jsoniter "github.com/json-iterator/go"
+)
+
+func marshalJSONIsEmpty(ptr unsafe.Pointer) bool {
+	return false
+}
+
+// from https://github.com/prometheus/prometheus/blob/main/util/jsonutil/marshal.go
```

Comment:
I would remove this comment, because I think it would be a good next step to import this method in prometheus/prometheus and use this method from common instead, after which this comment will be invalid.